test/test_transforms_v2_consistency.py

```diff
@@ -98,6 +98,8 @@ def __init__(
             ArgsKwargs((29, 32), antialias=False),
             ArgsKwargs((28, 31), antialias=True),
         ],
+        # rtol=atol=1 due to Resize v2 is using native uint8 interpolate path for biliear and nearest modes
+        closeness_kwargs=dict(rtol=1, atol=1),
```

Comment:
shouldn't this be `rtol=0, atol=1`?

Reply:
There is a case with the following error if run locally:
```
FAILED test/test_transforms_v2_consistency.py::test_call_consistency[Resize-12]

Mismatched elements: 382 / 11136 (3.4%)
Greatest absolute difference: 103 at index (1, 1, 14, 19) (up to 1 allowed)
Greatest relative difference: 1.0 at index (0, 0, 14, 10) (up to 0 allowed)
```